src/App.tsx

```diff
@@ -1,9 +1,51 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+type Movie = {
+  imdbId: string;
+  title: string;
+  description: string;
+  imdbUrl: string;
+  imgUrl: string;
+};
+
+type FilterOptions = {
+  query: string;
+};
+
+function getFilteredMovies(movies: Movie[], { query }: FilterOptions): Movie[] {
+  let filteredMovies = [...movies];
```

Comment:
According to the checklist, if you are using a non-mutating array method like `filter`, you don't need to create a copy of the array. You can directly filter the `movies` array without spreading it into `filteredMovies`.